Dangerfile.df.kts

```diff
@@ -0,0 +1,54 @@
+@file:DependsOn("io.github.ackeecz:danger-kotlin-commit-lint:0.1.0")
+@file:DependsOn("io.github.ackeecz:danger-kotlin-detekt:0.1.4")
+
+import systems.danger.kotlin.*
+
+import io.github.ackeecz.danger.commitlint.*
+import io.github.ackeecz.danger.detekt.DetektPlugin
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.function.BiPredicate
+import java.util.stream.Collectors
+
+register plugin CommitLintPlugin
+register plugin DetektPlugin
+
+danger(args) {
+
+    // GitHub checks 
+    val allSourceFiles = git.modifiedFiles + git.createdFiles
+    val changelogChanged = allSourceFiles.contains("CHANGELOG.md")
+    val sourceChanges = allSourceFiles.firstOrNull { it.contains("src") }
+
+    onGitHub {
+        val isTrivial = pullRequest.title.contains("#trivial")
+
+        // Changelog update check
+        if (!isTrivial && !changelogChanged && sourceChanges != null) {
+            warn("Any changes to library code should be reflected in the Changelog.\n\nPlease consider adding a note there and adhere to the [Changelog Guidelines](https://github.com/Moya/contributors/blob/master/Changelog%20Guidelines.md).")
```

Comment:
we can't apply this trivial rule for us, it's `WIP` or draft PR, we don't use the term `trivial` in PR. 
Also the message is ref to another repo

Reply:
@sleepylee should we still keep the Changelog check? The message is for providing the guidelines of how to write a Changelog actually 🙏

Reply:
@minhnimble I don't think we need it. We don't manage changelog in this way nor asking for developers to file in the changelog update on every push...it's not necessary. We could have various PRs to serve 1 feature update, to pull a release changelog I'd rather pull all from git history recursively.